functions

```diff
@@ -808,7 +811,7 @@ run_build_hook() {
     fi
 
     if ! declare -f build >/dev/null; then
-        error 'Hook '$script' has no build function'
+        error 'Hook '"$script"' has no build function'
```

Comment:
```suggestion
        error "Hook $script"' has no build function"
```

Reply:
The `error` function uses `prinft`, so this will work:

```suggestion
        error 'Hook %s has no build function' "$script"
```